pkg/chain/bitcoin/config.go

```diff
@@ -1,7 +1,33 @@
 package bitcoin
 
+import (
+	"fmt"
+
+	"github.com/btcsuite/btcd/chaincfg"
+)
+
 // Config stores configuration related to recovering BTC from a closed keep.
 type Config struct {
 	BeneficiaryAddress string
 	MaxFeePerVByte     int32
+	ChainName          string
+}
+
+// ChainParams parses the net param name into the associated chaincfg.Params
+func (c Config) ChainParams() (*chaincfg.Params, error) {
+	switch c.ChainName {
+	case "mainnet":
+		return &chaincfg.MainNetParams, nil
+	case "regtest":
+		return &chaincfg.RegressionNetParams, nil
+	case "simnet":
+		return &chaincfg.SimNetParams, nil
+	case "testnet3":
+		return &chaincfg.TestNet3Params, nil
+	case "":
```

Comment:
We could combine two cases.

```suggestion
	case "mainnet", "":
```

Reply:
that's a handy one!